pkg/api/types.go

```diff
@@ -385,6 +388,9 @@ const (
 	// BetaStorageClassAnnotation represents the beta/previous StorageClass annotation.
 	// It's currently still used and will be held for backwards compatibility
 	BetaStorageClassAnnotation = "volume.beta.kubernetes.io/storage-class"
+	// AlphaStorageTopologyAnnotation is set in the StorageClass and gives the scheduler
+	// a hint that the volumes in this StorageClass have a topology constraint
+	AlphaStorageTopologyAnnotation = "storage.alpha.kubernetes.io/topology-key"
```

Comment:
I'm going to remove this annotation from this PR for now because there is still more discussion on this topic that we have to do, and I don't want it to block everything else.  Let's use #43504 to continue the discussion.